draft-kwbdgrr-tsvwg-net-collab-rqmts.md

```diff
@@ -407,6 +411,25 @@ Requirement:
   REQ-SIGNAL-EXPOSURE-FAIRNESS:
   : Means to expose the signal independent of the application should be considered. An example of such exposure is OS APIs.
 
+## Exposure Handling {#exposure-handling}
+
+Signaling to the network ({{host-network}}, {{server-network}}) and consuming the signals from the network ({{network-host}}) will need to be facilitated by Application Programming Interfaces (APIs) for any application to use them. Signaling and retrieving of signals may not be performed at a single layer. Hence, a framework is required to abstract the underlying protocol(s) and allow the application(s) to retrive/send signals using a single or a set of API(s).
```

Comment:
```suggestion
Signaling to the network ({{host-network}}, {{server-network}}) and consuming the signals from the network ({{network-host}}) will need to be facilitated by Application Programming Interfaces (APIs) for any application to use them. Signaling and retrieval of the signals may not be performed at a single layer (although not encouraged). Hence, a framework is required to abstract the underlying protocol(s) and allow the application(s) to retrieve/send signals using a single or a set of API(s) independent of the channels that are used to convey the signals.  This APIs is required even if one single channel is used so that any application can consume the signals.
```